vmdb/app/models/vm_or_template.rb

```diff
@@ -329,24 +307,14 @@ def refresh_state
   end
 
   def run_command_via_parent(verb, options = {})
+    raise "VM/Template <#{name}> with Id: <#{id}> is orphaned." unless self.ext_management_system
```

Comment:
This message should say that the VM "is not associated with a Provider" which is what the original code was checked.  For orphaned you would want to call the `orphaned?` method and would also need to check for archived, but again that is not what it was originally checking.